docs/reuse/tutorial/setup_edge.rst

```diff
@@ -0,0 +1,90 @@
+First, install Multipass.
+
+.. seealso::
+
+    See more: `Multipass |
+    How to install Multipass <https://multipass.run/docs/install-multipass>`_
+
+Use Multipass to launch an Ubuntu VM with the name ``charm-dev``
+from the 24.04 blueprint:
+
+.. code-block:: bash
+
+    multipass launch --cpus 4 --disk 50G --memory 4G --name charm-dev 24.04
```

Comment:
I just noticed this, which is probably not how you intend it to look:

![image](https://github.com/user-attachments/assets/f6bca0b8-5d99-483d-a10f-e6be55beac39)
Changing to:

```suggestion
.. code-block:: terminal

    multipass launch --cpus 4 --disk 50G --memory 4G --name charm-dev 24.04
```

changes the highlighting (this is from a different branch which is why the text is different):

![image](https://github.com/user-attachments/assets/095e6957-6096-40e1-a49c-c878bac9261e)

Reply:
`terminal` falls back to no highlighting, as it's not a Pygments lexer:

```code
WARNING: Pygments lexer name 'terminal' is not known
```

Other tools that use Pygments knows how to read it correctly with `bash`. A sample from VS Code:

![image](https://github.com/user-attachments/assets/61660abe-2982-4973-be04-0a83f06b5119)

I'm blaming our doc stack. If the issue is really bothersome we can set the lexer to `text` for those odd lines. It's not highlighting much, anyway.